libraries/rush-lib/src/logic/incremental/InputsSnapshot.ts

```diff
@@ -122,7 +122,9 @@ const { hashDelimiter } = RushConstants;
  */
 export interface IInputsSnapshot {
   /**
-   * The raw hashes of all tracked files in the repository.
+   * The raw hashes of the files relevant to the projects we care about are stored.
+   * (e.g. when running `rush build`, the hashes of all tracked files in the repository are stored)
+   * (e.g. when running `rush build --only`, only the hashes of files under the specified project are stored)
```

Comment:
This won't work. Computation of operation hashes depends on the entire tree of their dependencies, _whether or not you are currently executing said dependencies_. So at minimum we _always_ need the expansion of `--to`, not just the values passed to `--only` to be able to determine build cache entry ids.

Reply:
I see. I just want to express that we are no longer storing all the hashes. I’ve removed that description.